docs/convolution/using.rst

```diff
@@ -65,27 +69,37 @@ interpolated using the kernel::
     >>> convolve([1, 4, 5, 6, np.nan, 7, 8], [0.2, 0.6, 0.2], boundary='extend')  # doctest: +FLOAT_CMP
     array([1.6 , 3.6 , 5.  , 5.75, 6.5 , 7.25, 7.8 ])
 
+..
+  EXAMPLE END
+
+..
+  EXAMPLE START
+  Constructing Arrays as Lists
```

Comment:
I think this should be Constructing Kernels from Lists

Reply:
Thanks for the suggestion!